docs/main.css

```diff
@@ -1,4 +1,5 @@
 /* SPDX-License-Identifier: MIT */
+/* SPDX-FileCopyrightText: 2022 Mercedes-Benz Tech Innovation GmbH */
```

Comment:
I do not think we need this here. We have the MIT license - that is enough. We have not embedded inside the CSS any images/svgs etc.

Reply:
Please remove copyright information - we use MIT license here and so it is not necessary.